README.md

```diff
@@ -150,6 +142,89 @@ Version:1
 $ 
 ```
 
+### List Processes Command
+```sh
+$ process-management list --help
+Usage: process management list [options]
+
+A command for listing all active process flows
+
+Options:
+  -h, --host <host>  substrate blockchain host address or FQDM, default - "localhost" (default: "localhost")
+  -p, --port <port>  specify host port number if it is not a default, default - 9944 (default: "9944")
+  --active           returns only active process flows
+  --disabled         returns only disabled process flows
+  --print            print debugging info
+  --help             display help for command
+
+#
+# examples
+#
+
+# --active
+$ process-management list --active 
+[
+  {
+    id: '0x732e69d9bee930b67c907ac97ef0deeac8e52635c16b266071d2f42211f485a1c8e677ed0f6c7a01b80a29e829baa5ee446d6f636b5f6163636570745f6f72646572d82c12285b5d4551f88e8f6e7eb52b8101000000',
+    version: 0,
+    initialU8aLength: 13,
+    registry: {},
```

Comment:
maybe these keys can be removed because they're not useful to the user

Reply:
Leave it for now and create another ticket to remove them.

Reply:
marking as resolved because this is not an AC